<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditPredefinedIP</class>
 <widget class="QDialog" name="DlgEditPredefinedIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="LabelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelIPaddress">
       <property name="text">
        <string>IP address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="EditIPaddress">
       <property name="font">
        <font>
         <family>Lucida Console</family>
        </font>
       </property>
       <property name="inputMask">
        <string>009.009.009.009</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelNetworkMask">
       <property name="text">
        <string>Network mask:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="EditNetworkMask">
       <property name="font">
        <font>
         <family>Lucida Console</family>
        </font>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelGateway">
       <property name="text">
        <string>Gateway:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="EditGateway">
       <property name="font">
        <font>
         <family>Lucida Console</family>
        </font>
       </property>
       <property name="inputMask">
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <spacer name="HLayoutSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
